Jenkinsfile

```diff
@@ -175,6 +175,7 @@ pipeline {
                         sh "echo STAN_OPENCL=true>> make/local"
                         sh "echo OPENCL_PLATFORM_ID=0>> make/local"
                         sh "echo OPENCL_DEVICE_ID=${OPENCL_DEVICE_ID}>> make/local"
+                        sh "make -j${env.PARALLEL} test-headers"
```

Comment:
[optional] Maybe OpenCL headers tests should nrun in parallel with existing headers tests.

Reply:
Yeah, that may be better. Will do.